test/cpp/jit/test_lite_interpreter.cpp

```diff
@@ -482,8 +482,23 @@ TEST(LiteInterpreterTest, GetByteCodeVersion) {
   auto version_v4 = _get_model_bytecode_version(test_model_file_v4);
   AT_ASSERT(version_v4 == 4);
 }
+
 #endif // !defined(FB_XPLAT_BUILD)
 
+TEST(LiteInterpreterTest, GetContainTypes) {
+  Module m("m");
+  m.define(R"(
+    def forward(self):
+      return 3
+  )");
+
+  std::stringstream ss;
+  m._save_for_mobile(ss, {}, true);
+
+  auto contained_types = _get_model_contained_types(ss);
+  AT_ASSERT(contained_types.size() >= 0);
```

Comment:
Could we test the actual type is expected?

Reply:
I thought about it. I'm a bit hesitate because the type might be changed depending on inline/optimization. Do you think the updated test `test/cpp/jit/test_mobile_type_parser.cpp` in https://github.com/pytorch/pytorch/pull/63130 is sufficient?